tests/filesystem.go

```diff
@@ -65,8 +65,6 @@ func prepareRootPartitionForPasswd(t *testing.T, partitions []*types.Partition)
 				{"/lib64/ld-linux-x86-64.so.2", filepath.Join(p.MountPath, "usr", "lib64")},
 				{"/lib64/libnss_files.so.2", filepath.Join(p.MountPath, "usr", "lib64")},
 				{"bin/amd64/id", filepath.Join(p.MountPath, "usr", "bin")},
```

Comment:
Should this be updated to copy from to `distro.IdCmd()`? If so then should we also update the stub for `id` to `id-stub` and add the related `GLDFLAGS`?

Reply:
We'd need to update this function to include the full filepath (not just the directory) so we could do the rename, but I'm not opposed.

Reply:
Ah, I'm fine either way. My thinking was that it'd be nice to not have it just looking in a relative path for the binary to copy.

Reply:
I think I misunderstood this actually. There's two parts here. I was referring to the destination it's copied to which should be the same as `distro.IdCmd()`. There's also the issue of the source, which, as per my comment in `build_blackbox_tests`, is probably bin/amd64/id but really should be determined by `which` to account for the current PATH. I'm in favor of just leaving this for now; I'm not sure there's a huge benefit. 

Come to think of it, since we don't need `user{mod,add}`, we should be able to drop the .so's too, yes?